charts/seed-monitoring/charts/alertmanager/templates/alertmanager.yaml

```diff
@@ -33,6 +33,7 @@ spec:
   selector:
     component: alertmanager
     role: monitoring
+{{- if .Values.emailConfigs }}
```

Comment:
Any reason why all the other resources are applied (services, configs, etc.) although the AlertManager itself is not deployed?

Reply:
I agree. The services and configs should not be deployed unless I am missing something.